### ISSUE TITLE
Change width for links in Icon List Widget

### ISSUE THREAD

**Issue by fxaviers** [elementor/elementor#11945]
## Prerequisites

- [x] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [x] The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )

**What problem is your feature request going to solve? Please describe.**
By creating a list of icons with a link, the link is added across the width, not just the icon and text.
This generates false clicks, the visitor can click on an area without content but leaves the page.

**Describe the solution you'd like**
This can be solved with CSS, but if Elementor already creates a style for the links, then it could be changed, without needing a custom code.
![image](https://user-images.githubusercontent.com/17536232/88338331-95855180-cd0e-11ea-8ddf-3e30853c667e.png)

![image](https://user-images.githubusercontent.com/17536232/88338349-9ae29c00-cd0e-11ea-8f38-b549103f8f0e.png)

**Describe alternatives you've considered**
```
.elementor-widget .elementor-icon-list-item a {
    width: auto;
}
```

**Comment by fabianenzensperger** [comment]
Yes yes yes, an icon or svg logo has tooooooo much linkable white space around the widget.. Good to see it changes with 3.0?

**Comment by shilo-ey** [comment]
This issue has been resolved in Elementor v3.0

Feel free to update

Thanks!
